README.md

```diff
@@ -193,50 +193,45 @@ Configuration is handled differently depending on how the connector is installed
+| client.proxy.password             | The client proxy password. Requires `client.proxy` to be set to `true`.                                                                                                                                                         | *String*  ||
+| client.proxy.port                 | The client proxy port. Requires `client.proxy` to be set to `true`.                                                                                                                                                             | *Integer* ||
+| client.proxy.pref.auth.type       | The authentication type to use with the client proxy. Must be one of `BASIC`, `DIGEST` or `X_ABLY_TOKEN`. Requires `client.proxy` to be set to `true`.                                                                          | *String*  | Basic   |
+| client.push.full.wait             | Sets whether Ably should wait for all the effects of push REST requests before responding.                                                                                                                                      | *Boolean* ||
+| client.tls                        | Sets whether TLS is used for all connection types.                                                                                                                                                                              | *Boolean* | True    |
+| client.token.params               | Sets whether the configured token parameters are used.                                                                                                                                                                          | *Boolean* ||
+| client.token.params.capability    | Stringified JSON capability requirements for the token. When omitted, the REST API default to allow all operations is applied by Ably, with the string value `{“*”:[“*”]}`. Requires `client.token.params` to be set to `true`. | *String*  ||
+| client.token.params.client.id     | The client ID to include with the token. Requires `client.token.params` to be set to `true`.                                                                                                                                    | *String*  ||
+| client.token.params.ttl           | The requested time to live (TTL) for the token in milliseconds. When omitted, the REST API default of 60 minutes is applied by Ably. Requires `client.token.params` to be set to `true`.                                        | *Boolean* | 0       |
+| client.loglevel                   | Sets the verbosity of logging.                                                                                                                                                                                                  | *Integer* | 0       |
```

Comment:
Might it be useful to include example integer values for WARN/INFO/DEBUG (or link to where they can be found)?

Reply:
Again, this is just the existing functionality, but it is a fair shout. It actually took me a while to go grok the ably-java code and figure out the magic number you need to pass in for verbose logging here is in fact `2`. I'll raise a ticket to improve this later..